pkg/core/storage/boltdb_store.go

```diff
@@ -71,6 +71,14 @@ func (s *BoltDBStore) Get(key []byte) (val []byte, err error) {
 	return
 }
 
+// Delete implements the Store interface.
+func (s *BoltDBStore) Delete(key []byte) error {
```

Comment:
hm. Are we sure we want to allow this func to be public?

Reply:
We have no other choice, unfortunately.